src/registrar/templatetags/field_helpers.py

```diff
@@ -152,11 +156,16 @@ def input_with_errors(context, field=None):  # noqa: C901
     if group_classes:
         context["group_classes"] = " ".join(group_classes)
 
+    # We handle sublabel_text here instead of directy in the template to avoid conflicts
+    if sublabel_text:
+        sublabel_div_id = f"{attrs['id']}__sublabel"
+        described_by.insert(0, sublabel_div_id)
+
     if described_by:
         # ensure we don't overwrite existing attribute value
         if "aria-describedby" in attrs:
             described_by.append(attrs["aria-describedby"])
-        attrs["aria_describedby"] = " ".join(described_by)
```

Comment:
This was a seondary / unrelated bug that broke, eg, the domain manager form input